CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -1101,14 +1101,15 @@ void sprite_sheet::draw_sprite(render_target* pCanvas, size_t iSprite, int iX,
     // Use the target cycle length (15 cycles) to calculate the angle from 0 to
     // 2*pi.
     int currentVariation = static_cast<int>(
-        sin(static_cast<double>(game_ticks % 15) / 15.0 * 2 * pi) * 50);
+        sin(static_cast<double>(effect_ticks % 15) / 15.0 * 2 * pi) * 50);
     int err = SDL_SetTextureColorMod(pTexture, 0, 205 + currentVariation, 0);
     if (err < 0) {
       throw std::runtime_error(SDL_GetError());
     }
   }
 
-  if (effect == animation_effect::jelly) {
+  // TODO: Adjust jelly frequency and duration.
+  if (effect == animation_effect::jelly && effect_ticks % 200 < 30) {
```

Comment:
200 and 30 are better off as constants. jelly_effect_period, and jelly_effect_duration perhaps.

Reply:
Done, also updated these to be similar to what i observed in the original.